### ISSUE TITLE
Handle high cardinality span names

### ISSUE THREAD

**Issue by makeavish** [SigNoz/signoz#4654]
According to Otel spec span names MUST not contain high cardinal IDs. But some users accidentally make instrumentation mistakes which causes high cardinal span names.
It breaks service list on SigNoz home page. There should in product handling of such errors and guide user to fix their instrumentation.

**Comment by ankitnayan** [comment]
@srikanthccv did we plan for anything more when we shipped the last solution?

**Comment by srikanthccv** [comment]
No. The next remaining step is updating SDKs to send the data for https://github.com/SigNoz/signoz-otel-collector/pull/296
